karma.conf.js

```diff
@@ -52,7 +56,6 @@ module.exports = () => {
       jasmine: {
         random: false,
       },
-      clearContext: false, // leave Jasmine Spec Runner output visible in browser
```

Comment:
Removing this configuration to address the "Some of your tests did a full page reload!" error introduced in Chrome 128. We don't use Jasmine Spec Runner, so we can safely use the default.
See: https://github.com/angular/angular-cli/issues/28271#issuecomment-2319380288